x/concentrated-liquidity/tick_test.go

```diff
@@ -409,40 +409,100 @@ func (s *KeeperTestSuite) TestGetTickInfo() {
 func (s *KeeperTestSuite) TestCrossTick() {
 	var (
 		preInitializedTickIndex = DefaultCurrTick.Int64() - 2
+		expectedUptimes         = getExpectedUptimes()
+		emptyUptimeTrackers     = wrapUptimeTrackers(expectedUptimes.emptyExpectedAccumValues)
 		defaultAdditiveFee      = sdk.NewDecCoinFromDec(USDC, sdk.NewDec(1000))
 	)
 
 	tests := []struct {
 		name                         string
 		poolToGet                    uint64
+		preInitializedTickIndex      int64
 		tickToGet                    int64
+		initGlobalUptimeAccumValues  []sdk.DecCoins
+		globalUptimeAccumDelta       []sdk.DecCoins
+		expectedUptimeTrackers       []model.UptimeTracker
 		additiveFee                  sdk.DecCoin
 		expectedLiquidityDelta       sdk.Dec
 		expectedTickFeeGrowthOutside sdk.DecCoins
 		expectedErr                  bool
 	}{
 		{
-			name:                         "Get tick info on existing pool and existing tick",
+			name:                    "Get tick info on existing pool and existing tick below current tick (nonzero uptime trackers)",
```

Comment:
nit: let's remove on "existing pool" part from every test case name, please. It adds cognitive overhead while being unimportant context relative to other test case details because pool existence check happens once at the top level

Reply:
Agreed, done